tools/GenerateMergeOptionsMethods/GenerateMergeOptionsMethods.csproj

```diff
@@ -32,7 +32,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.41.0" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="4.40.0" />
```

Comment:
moving  back to 4.40?

Reply:
lol. nice catch. VS had an error and it reverted back to 4.40 and I didn't notice. I have re-tested